docs/handbook/getting-started.md

```diff
@@ -585,6 +585,12 @@ Replace `X` with the appropriate letter for your USB device.
 sudo dd bs=4M if=./image.usb of=/dev/sdb status=progress && sync
 ```
 
+<!-- NOTE: --> <i class="fa fa-info-circle fa-lg" aria-hidden="true"></i> **NOTE:**
+<div class="well">
+Recent releases of OpenIndiana Hipster use the GNU version of dd (/usr/gnu/bin/dd), which should support the `status=progress` option.
+Older releases however may still be using illumos dd (/usr/bin/dd) which may not support this option.
```

Comment:
This is wrong. The issue is not in "old releases", but in PATH setting. A lot of users like to set /usr/gnu/bin to the end of PATH, as GNU tools doesn't always work in proper way on illumos/Solaris (for example, doesn't support ZFS ACL).